examples/kubernetes_orchestration/steps/skew_comparisons.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+import pandas as pd
+
+from zenml.steps import Output, step
+
+
+@step
```

Comment:
This also relates to your two code suggestions above. You're totally right, this step feels pretty useless as is.

I see four possible solutions for this:
1. Include the post execution code of facets in run.py to actually load up the skew comparison at the end of run.py. - Personally my preferred option at the moment, even though it feels not super clean either.
2. Rename the skew_comparison step and remove facets as dependency - Personally not a big fan of this option, because what is that step doing then?
3. Remove the skew_comparison step and facets as dependency in favor of another step that can demonstrate the parallel execution - personally I don't have any ideas here that I like more than the current step
4. Use a different example pipeline altogether - I was planning to just copy over the quickstart example and run that on kubernetes, but that doesn't work yet since the k8s orchestrator only works with remote stacks atm, which means it is not compatible with mlflow deployer.

As mentioned, I'm personally leaning towards simply including the facets post execution in run.py for now, and then changing the example to run quickstart once local k8s is supported.

Any suggestions/ideas?

Reply:
Mhm my personal preference would be to get rid of facets entirely actually, just to keep the example as simple as possible. We might get people that are really interested in trying the kubernetes orchestrator but have never heard/don't care about facets at all and then have to bother to remove it.
However I don't have any good suggestions for steps that could replace the skew comparison either, but we're also not showcasing parallel execution in any other orchestrator example. So IMO we could even get rid of it without replacement.

But in general all options are acceptable to me, so I'll let you choose whichever one you like best unless some other reviewers have stronger opinions on this.

Reply:
IMO we should show parallel execution where applicable, it's cool! 

Long term I think changing the orchestrator examples to run quickstart pipeline would be the right move. Let's do that next time we rework the quickstart.

As a temporary fix I would still prefer 1) for now, but I'll wait and leave the discussion open for other reviewers.

Reply:
I will go for 1) since that also tests that the post execution code works with the kubernetes metadata store (which it might currently not as @stefannica pointed out in his review).